tests/patterns/dap.py

```diff
@@ -11,7 +11,7 @@
 from tests.patterns import some, _impl
 
 
-id = some.int.in_range(0, 10000)
+id = some.int.in_range(-10000, 10000)
```

Comment:
ids seem to be negative in the new code

Reply:
Yep, that's intentional, to distinguish synthetic thread IDs from OS-provided ones. I also found that using negative IDs makes it easier to distinguish from other numeric values in JSON logs, which tend to be positive (counts, user data etc). Thank you for catching this one!